wagtail/documents/admin_urls.py

```diff
@@ -10,8 +10,8 @@
         documents.IndexView.as_view(results_only=True),
         name="index_results",
     ),
-    path("add/", documents.add, name="add"),
-    path("edit/<int:document_id>/", documents.edit, name="edit"),
+    path("add/", documents.CreateView.as_view(), name="add"),
```

Comment:
If I'm not mistaken, this view is not used at all anymore. Shall we deprecate it instead?

Reply:
I'd lean towards keeping it, but that's probably more out of an aesthetic desire for keeping things consistent/symmetrical with other areas of the admin (which might be more meaningful if/when we convert this to a viewset) than any compelling technical reason.